CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 CHANGELOG
 =========
+1.4.0 (June, 2024)
+------
+- Support for executing DAST Scans via AppScan 360°.
+- Support for Personal Scan.
```

Comment:
Support Personal scans in HCL AppScan on Cloud.

Reply:
Adding "Support for personal scans in HCL AppScan on Cloud and HCL AppScan 360°".